frontend/app/login/page.tsx

```diff
@@ -1,8 +1,106 @@
-import React from "react"
+"use client"
+import Typography from '@mui/material/Typography'
+import Grid from '@mui/material/Grid'
+import TextField from '@mui/material/TextField'
+import Button from '@mui/material/Button'
+import { createTheme, ThemeProvider } from '@mui/material/styles'
+import { useState } from 'react'
+import { LoginValidation } from '../helpers/login.helper'
 
-const Login = (a: number): React.ReactElement => {
-  console.log("Hi")
-  return <h1>Login Page</h1>
+const Login = (): React.ReactElement => {
+  interface UserCredentials {
+    username: string
+    password: string
+  }
+  const defaultCredential: UserCredentials = {
+    username: "",
+    password: ""
+  }
+  interface ValidationError {
+    username: string
+    password: string
+  }
+  const defaultValidationError: ValidationError = {
+    username: "",
+    password: ""
+  }
+  const [credentials, setCredentials] = useState<UserCredentials>(defaultCredential)
+  const [loginFormValidationError, setLoginFormValidationError] = useState<ValidationError>(defaultValidationError)
+  const [errorReceived, setErrorReceived] = useState<ValidationError>(defaultValidationError)
+
```

Comment:
Can we rename loginFormValidationError to validationError? No need to att loginForm to the variable since the variable is already in the login page.